app/src/main/java/hous/release/android/presentation/login/LoginActivity.kt

```diff
@@ -27,8 +29,36 @@ class LoginActivity : BindingActivity<ActivityLoginBinding>(R.layout.activity_lo
         initKakaoLoginBtnClickListener()
         initIsSuccessKakaoLoginObserver()
         initIsInitUserInfoObserver()
-        initIsSucessLoginObserver()
         initBackPressedCallback()
+        initIsJoiningRoomObserve()
+        initIsUserObserve()
+    }
+
+    private fun initIsUserObserve() {
+        loginViewModel.isJoiningRoom.observe(this) {
+            if (loginViewModel.isJoiningRoom.value == true) {
```

Comment:
```suggestion
            if (loginViewModel.isJoiningRoom.value) {
```
변수가 boolean이라 `== true`를 제거해도 됩니다

Reply:
> 변수가 boolean이라 `== true`를 제거해도 됩니다

<img width="580" alt="image" src="https://user-images.githubusercontent.com/84129098/197099566-ca3eb514-0674-4d8d-8c9e-165f06c7949f.png">

요로코롬 떠서 명시해줬습니다ㅏ

Reply:
아ㅋㅋㅋ 라이브 데이터 킹받네... 확인임다~

Reply:
> 아ㅋㅋㅋ 라이브 데이터 킹받네... 확인임다~

얼른 플로우 배워올게요 ㅎㅎ..